tei/la-hija-del-aire-ii.xml

```diff
@@ -3388,16 +3388,10 @@
             <l>mi felicidad en vella,</l>
             <l>y porque voy a buscalla</l>
             <l>con ánimo de creella.</l>
-
           </sp>
           <stage>Vanse.</stage>
         </div>
-
-
-
-      </div>
-      <div type="act" n="1">
-        <div type="scene" resp="#rojas" n="1">
+        <div type="scene" resp="#rojas" n="12">
           <stage>Vanse.</stage>
```

Comment:
Unless I miss something, this `<stage>` is not in the digital source, see https://www.cervantesvirtual.com/obra-visor/la-hija-del-aire-segunda-parte--0/html/ff92d762-82b1-11df-acc7-002185ce6064_4.html. We should probably use the opportunity to remove it.

Reply:
I could have a look at this, and happily I can say that my script is not responsible for splitting the act II into another act. This was already in the original source file that I run muy script :) In any case, it is an evident error that has to be corrected. So thanks Jan!